config/locales/provider_interface/make_offer.yml

```diff
@@ -87,3 +87,6 @@ en:
             base:
               identical_to_existing: The new offer is identical to the current offer
               different_ratifying_provider: The offered course's ratifying provider must be the same as the one originally requested
+              other_offer_already_accepted: You cannot make an offer because the candidate has already accepted one
+              application_rejected_by_default: You cannot make an offer because the application has been automatically rejected
+              only_latest_application_rejection_can_be_reverted_on_apply_2: You cannot make an offer because you can only do so for the last application you rejected
```

Comment:
the wording here assumes the provider is making an offer on a rejected application, they might just be making an offer without reverting a rejection?

Reply:
Spoke to design and waiting on receiving a response about this.

Reply:
updated